daemon/logger/journald/read.go

```diff
@@ -220,10 +220,6 @@ func (s *journald) readLogs(logWatcher *logger.LogWatcher, config logger.ReadCon
 	}
 	// Add a match to have the library do the searching for us.
 	cmatch = C.CString("CONTAINER_ID_FULL=" + s.vars["CONTAINER_ID_FULL"])
```

Comment:
I suppose it just panics according to https://github.com/golang/go/commit/647eaed93b2aba07d4d4d032d5f58bf6dbc06e5a